tick/optim/prox/prox_binarsity.py

```diff
@@ -0,0 +1,126 @@
+# -*- coding: utf8 -*-
+
+import numpy as np
+from .base import Prox
+from .build.prox import ProxBinarsity as _ProxBinarsity
+
+
+class ProxBinarsity(Prox):
+    """Proximal operator of binarsity, namely total-variation
+    regularization followed by centering within sub-blocks of the
+    coefficients. Blocks (non-overlapping) are specified by the
+    ``blocks_start`` and ``blocks_length`` parameters.
```

Comment:
Do you check anywhere in the code that the blocks are indeed not overlapping each other?